addons/website/static/src/js/utils.js

```diff
@@ -4,7 +4,16 @@ import { intersection } from "@web/core/utils/arrays";
 import { csrf_token, _t } from "@web/legacy/js/services/core";
 import { renderToElement } from "@web/core/utils/render";
 
-
+let _languages;
```

Comment:
I chose a cache with an expiration of 10 seconds. Just to avoid a multiple request when _computeWidgetVisibility is called on all the snippets options.